provider/go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/hashicorp/terraform-provider-aws v1.60.1-0.20220923175450-ca71523cdc36
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pulumi/providertest v0.1.3
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.94.0
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.95.0
```

Comment:
This is incredibly suspect, we probably should have an issue tracking this. Upstream upgrade automation should not upgrade the bridge. We want to decouple changes as much as possible so we can bisect issues easily.

Reply:
Yeah! I definitely shouldn't do both. Upgrading the bridge to 3.95.0 was already merged, so we don't have an issue here. Just need to rebase the PR

Will cut an issue in upgrade-provider about this

Reply:
I have cut an issue in ci-mgmt already.

Reply:
https://github.com/pulumi/ci-mgmt/issues/1141 - I think upgrade-provider can do this, but ci-mgmt is not asking it politely.

Reply:
Yeah, just found the same: https://github.com/pulumi/ci-mgmt/blob/cf231765a6042eea3c4f9de32c5ecb08f23696b8/provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/upgrade-provider.yml#L83

It should be configured to only upgrade the provider and not all

Reply:
Just rebased to remove the bridge upgrade changes from the diff